pnpm-lock.yaml

```diff
@@ -41542,6 +41425,7 @@ packages:
 
   /path-dirname@1.0.2:
     resolution: {integrity: sha512-ALzNPpyNq9AqXMBjeymIjFDAkAFH06mHJH/cSBHAgU0s4vfpBn6b2nf8tiRLvagKD8RbTpq2FKTBg7cl9l3c7Q==}
+    requiresBuild: true
```

Comment:
This feels funny here but no idea what it is.

Reply:
Yeah...  It might be one of those packages that has a native node module for MacOS, but not other platforms.

I'm going to try reverting it by hand and see if CI is still happy.